api_v3/services/ServerNodeService.php

```diff
@@ -192,17 +191,21 @@ private function addNewServerNode(KalturaServerNode $serverNode)
 	}
 
 	/**
-	 * Unregister server node by id
+	 * Mark server node offline
 	 *
-	 * @action unregister
+	 * @action markOffline
 	 * @param string $serverNodeId
 	 * @throws KalturaErrors::INVALID_OBJECT_ID
 	 * @return KalturaServerNode
 	 * @throws KalturaAPIException
 	 */
-	function unregisterAction($serverNodeId)
+	function markOfflineAction($serverNodeId)
 	{
-		$dbServerNode = ServerNodePeer::retrieveByPK($serverNodeId);
+		$criteria = new Criteria(ServerNodePeer::DATABASE_NAME);
```

Comment:
Why aren't U using ServerNodePeer::retrieveActiveServerNode?

Reply:
I have the serverNodeId and don't have the hostname